docs/running-on-yarn.md

```diff
@@ -646,6 +646,16 @@ To use a custom metrics.properties for the application master and executors, upd
   </td>
   <td>0.9.0</td>
 </tr>
+<tr>
+  <td><code>spark.yarn.report.logging.frequency</code></td>
```

Comment:
Do we have other sister configurations with this? Otherwise, instead of introducing a new config namespace, `spark.yarn.report.logging.*`, shall we rename to `spark.yarn.report.loggingFrequency`?